controllers/webhook/status_updater.go

```diff
@@ -156,7 +162,6 @@ func (su StatusUpdater) completeRunConfiguration(
 				"Action",
 				"Get",
 			)
-			return nil
```

Comment:
Hmmm I don't get why we've removed the `return nil` and instead let this return an error. I also don't get why we haven't done the same thing with the not found error below when updating the status.

Reply:
we were explicitly ignoring the error if it was `not found` so it would just progress, the logic has been moved to the layer above with the changes in this pr

Reply:
see Handle function as above